### PR TITLE
Make E2E tests run on Functions V3 runtime

### PR DESCRIPTION
**Pull request by AnatoliB**
We'll need to propagate this change to both `v3.x/ps7` and `v3.x/ps6` branches.

### CODE REVIEW

File: test/E2E/TestFunctionApp/extensions.csproj [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+	<WarningsAsErrors></WarningsAsErrors>
```
Comment (Francisco-Gamino) [inline test/E2E/TestFunctionApp/extensions.csproj:4]: If we are not enabling `WarningsAsErrors`, should we remove it?

**Review by Francisco-Gamino — APPROVED** [review]
One minor comment. Other than that, LGTM.

File: test/E2E/TestFunctionApp/extensions.csproj (hunk 1 above)
Comment (AnatoliB) [reply test/E2E/TestFunctionApp/extensions.csproj:4]: This code was auto-generated by the `func extensions install` command, so I would suggest we leave it as is, even though we could technically improve it.

Comment (Francisco-Gamino) [reply test/E2E/TestFunctionApp/extensions.csproj:4]: Sounds good. Thanks for clarifying.

**Pull request merged**